lib/najax.js

```diff
@@ -92,9 +92,9 @@ function najax (uri, options, callback) {
   } else if (o.auth) {
     options.auth = o.auth
   }
+  /* pass keep-alive agent if provided */
+  if (o.agent) options.agent = o.agent
 
-  if (options.auth) o.auth = options.auth
-  if (options.agent) o.agent = options.agent
```

Comment:
These two lines have no effect. 

> if (options.auth) o.auth = options.auth
Not needed as handed already in the AUTHENTICATION block above

> if (options.agent) o.agent = options.agent
This should be the other way around.